layershell/layer.go

```diff
@@ -64,6 +64,11 @@ func SetLayer(window *gtk.Window, layer LayerShellLayerFlags) {
 	C.gtk_layer_set_layer(w, C.GtkLayerShellLayer(layer))
 }
 
+func SetNamespace(window *gtk.Window, name string) {
+	w := nativeWindow(window)
+	C.gtk_layer_set_namespace(w, name)
```

Comment:
```suggestion
	cName := C.CString(name)
	C.gtk_layer_set_namespace(w, cName)
	C.free(unsafe.Pointer(cName))
}
```

Reply:
this will need to be converted to a C string, which I think we'll actually need to manually free as well. (I'm pretty terrible at C)